ubi.repo

```diff
@@ -0,0 +1,71 @@
+[ubi-8-baseos-rpms]
```

Comment:
as a minimal number of packages are being added, do we need to define all these repos?

Reply:
yes, this is an autogenerated sowe don't want to change anything